prime-router/src/test/kotlin/serializers/Hl7SerializerTests.kt

```diff
@@ -1249,6 +1251,17 @@ OBX|3|DLN|53245-7^Driver license^LN||99999999^NJ|a^year^UCUM
         }
     }
 
+    @Test
+    fun `test HL7HapiErrorProcessor parse error codes exist for each type in Element Type enum`() {
```

Comment:
This will help remind us if we need to create a new error code if/when a new element type gets added